tests/data_validation/data_validation_checkbox_component.test.ts

```diff
@@ -28,6 +33,24 @@ describe("Checkbox in model", () => {
     });
   });
 
+  test("Checkbox are removed when clearing the content of the cell", () => {
```

Comment:
Also test that other rule types aren't removed if it's intended to work like that